src/platform/silabs/platformAbstraction/SilabsPlatformBase.h

```diff
@@ -56,6 +56,9 @@ class SilabsPlatformAbstractionBase
     virtual CHIP_ERROR FlashErasePage(uint32_t addr) { return CHIP_ERROR_NOT_IMPLEMENTED; }
     virtual CHIP_ERROR FlashWritePage(uint32_t addr, const uint8_t * data, size_t size) { return CHIP_ERROR_NOT_IMPLEMENTED; }
 
+    // soft reset
```

Comment:
We should use the standard comment formating whenever possible with the correct formatting

```suggestion
    /**
     * @brief Function trigger the platform to execute a software reset.
     *              Anything after this function will not be executed since the device will reboot.
     */
```